BUILD.bazel

```diff
@@ -46,10 +46,21 @@ go_binary(
 go_image(
     name = "bazel-remote-base",
     embed = [":go_default_library"],
+    goarch = "amd64",
+    goos = "linux",
```

Comment:
Just a minor issue: can you rearrange this so the original `bazel-remote-base` item is unchanged (`goos` and `goarch` after `static`), to make the diff smaller? And copy the same order for the new `bazel-remote-base-arm64` item.

Reply:
this is automatically formatted with `buildifier`. I would leave it as it is.

Reply:
The order is automatically formatted with `buildifier` in the bazel file. I would leave it as it is. What do you think?

Reply:
I'm not familiar with buildifier, but sounds ok to me.